transcom/mymove
# B 18392 prime counseled ppm aoa link

## Summary

Make the AOA download link available to a customer on Prime Counseled PPM shipments.  This should only be available once the Prime has completed counseling.  Note: this will not show on a PPM that does not have an advance requested.

### How to test

1. Login as a milmove customer.
2. Create a Prime Counseled PPM shipment.
3. Login as a TOO and approve the shipment.
4. Login as a Prime counselor and complete counseling for the move.
5. Log back in as the milmove customer and verify that you can see the link to download the AOA packet.
